hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -313,8 +313,7 @@ checkClose ctx parties initialUtxoHash sig cperiod headPolicyId =
 
   checkSnapshot
     | closedSnapshotNumber > 0 =
-      traceIfFalse "H11" $
-        verifySnapshotSignature parties closedSnapshotNumber closedUtxoHash sig
```

Comment:
BTW: This would have been already a breaking change to the scripts. Thought I mention it so you can build intuition.

Reply:
updating changelog.